### PR TITLE
fixing matlab wrapping

### PR DESCRIPTION
**Pull request by StevenPuttemans**
This fix resolves issue #1118

### CODE REVIEW

File: modules/matlab/include/opencv2/matlab/bridge.hpp [hunk 1]
```diff
@@ -522,6 +522,11 @@ class Bridge {
   Bridge& operator=(const Ptr_TonemapReinhard& ) { return *this; }
   Ptr_TonemapReinhard toPtrTonemapReinhard() { return Ptr_TonemapReinhard(); }
   operator Ptr_TonemapReinhard() { return toPtrTonemapReinhard(); }
+
+  // ---------------------------   Ptr_float   ----------------------
+  Bridge& operator=(const Ptr_float& ) { return *this; }
+  Ptr_float toPtrFloat() { return Ptr_float(); }
```
Comment (alalek) [inline modules/matlab/include/opencv2/matlab/bridge.hpp]: BTW, Looks like this method always create new empty "Ptr", so statement below will not use "input value" at all:
```
Ptr_float lowerBound = inputs[4].empty() ? (Ptr_float) Ptr<float>() : inputs[4].toPtrFloat();
```

**Comment by alalek** [comment]
:+1:

File: modules/matlab/include/opencv2/matlab/bridge.hpp (hunk 1 above)
Comment (StevenPuttemans) [inline modules/matlab/include/opencv2/matlab/bridge.hpp]: This is just to fix the build process. If they want to make sure the code is doing something useful, than that is for the guy of the other PR that wrapped this wrongfully xD

Comment (StevenPuttemans) [inline modules/matlab/include/opencv2/matlab/bridge.hpp]: But I get what you saying!

**Pull request merged**